### PR TITLE
Add Doctrine ORM and multicolumn filtration

### PR DESCRIPTION
**Pull request by MisatoTremor**

### CODE REVIEW

File: src/Knp/Component/Pager/Event/Subscriber/Filtration/Doctrine/ORM/QuerySubscriber.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?php
```
Comment (pilot) [inline src/Knp/Component/Pager/Event/Subscriber/Filtration/Doctrine/ORM/QuerySubscriber.php]: fix CS on file

Comment (MisatoTremor) [inline src/Knp/Component/Pager/Event/Subscriber/Filtration/Doctrine/ORM/QuerySubscriber.php]: Fixed

**Comment by pilot** [comment]
rebase pls

**Comment by MisatoTremor** [comment]
@pilot: Rebased and squashed

**Comment by pilot** [comment]
@MisatoTremor :+1: one thing can you add please to the usage doc some examples https://github.com/KnpLabs/knp-components/blob/master/doc/pager/usage.md

**Comment by MisatoTremor** [comment]
@pilot Done. Does this sound OK?

**Comment by pilot** [comment]
https://github.com/KnpLabs/knp-components/releases/tag/1.3.2

**Comment by MisatoTremor** [comment]
:)

**Pull request merged**